freeipa.spec.in

```diff
@@ -57,8 +57,12 @@ Source0:        freeipa-%{version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
 BuildRequires:  openldap-devel
+%if 0%{?fedora} > 25
+BuildRequires: krb5-devel >= 1.15-5
```

Comment:
@simo5 @frozencemetery unfortunately, the provide of "krb5-kdb-version = 6.1" is on krb5-libs, not on krb5-devel, so I cannot do a buildrequires dependency this way.